src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/RaCandidateRepository.php

```diff
@@ -163,13 +163,9 @@ public function createSearchQuery(RaCandidateQuery $query)
         $queryBuilder = $this->createQueryBuilder('rac');
 
         // Modify query to filter on authorization
-        $this->authorizationRepositoryFilter->filter($queryBuilder, $query->authorizationContext, 'rac.identityId', 'rac.institution', 'iac');
-
-        if ($query->actorInstitution) {
-            $queryBuilder
-                ->andWhere('rac.raInstitution = :raInstitution')
-                ->setParameter('raInstitution', $query->actorInstitution);
-        }
+        $queryBuilder
```

Comment:
The authorizationRepositoryFilter is no longer in use in the RaCandidateRepository, but you explicitly changed the `InstitutionAuthorizationRepositoryFilter` in this commit. I assume to fix an issue in the IdentityRepository or in one of the other repos created by the `AuthorizationRepositoryFactory`? I don't see cohesion yet.